source/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Electric Pomato", 
+    "icons": [
+        {
+            "src":"./img/pomatoPNG.png",
+            "sizes":"144x144",
+            "type":"image/png"
+        },
+        {
+            "src": "./img/green.svg", 
+            "type": "image/svg",
+            "sizes":"48x48 72x72 96x96 128x128 150x150 256x256 512x512 1024x1024"
+        }
```

Comment:
When I was running this locally, I seemed to be getting errors with the src paths for these icons. It seems to work for me when removing the './' so -> img/pomatoPNG.png for example.